assets/base.css

```diff
@@ -2154,9 +2154,9 @@ product-info .loading-overlay:not(.hidden) ~ *,
   scale: 0.9;
 }
 
-@media screen and (min-width: 990px) {
+@media screen and (min-width: 1090px) {
   .utility-bar__grid--3-col {
-    grid-template-columns: 3fr 4fr 3fr;
+    grid-template-columns: 1fr 1fr 1fr;
     grid-template-areas: 'social-icons announcements language-currency';
```

Comment:
This seems like a good fix. With all the icons present on the same line and not wrapping, it does seem a bit crowded however, I believe the number of merchants who will use all of the social icons is relatively small. The other point I wanted to bring up is that our previous break point coincided with when the mobile drawer would appear, the social icons and locales would become hidden and the drawer would appear at the same breakpoint. Now with the social icons/ locales becoming hidden and the mobile drawer appearing at different break points seems a bit odd to me.  This is not a blocker for the release but I did want to point this out and am wondering how other people feel about this change.

Reply:
I agree with Louisa that now we have a gap between 990px and 1090px when you can't reach social icons 🤔.